test.js

```diff
@@ -3,3 +3,9 @@ var foo = '' ;
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+var fooStag = '' ;
+
+if(fooStag  === "fooStag") {
+  console.log( 'fooStag' )
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.staging.codacy.org/gh/machadoit-test-org/eslint-test-stag/pullRequest?prid=13589)
```suggestion
  console.log( "fooStag" )
```